dags/libero/xml/__init__.py

```diff
@@ -0,0 +1,32 @@
+from typing import Iterable
+from xml.dom import XML_NAMESPACE
+
+from lxml.etree import ElementTree
+
+
+# namespaces
+XLINK_NS = 'http://www.w3.org/1999/xlink'
+
+# namespace_maps
+XLINK_MAP = {'xlink': XLINK_NS}
+
+# clark notation
+# reference: http://www.jclark.com/xml/xmlns.htm
+XLINK_HREF = '{%s}href' % XLINK_NS
+XML_BASE = '{%s}base' % XML_NAMESPACE
+
+
+
+def get_element_text_from_xpaths(xml: ElementTree, xpaths: Iterable[str], namespaces: dict = None) -> str:
```

Comment:
annoying to have to pass in the `namespaces`, but necessary

Reply:
agreed but that's part of lxml. There is a default value of None in the case where a namespace is not required for a query.